### PR TITLE
i18n(zh-cn): Update sidebar.mdx & configuration.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- Update sidebar.mdx & configuration.mdx

### CODE REVIEW

File: docs/src/content/docs/zh-cn/reference/configuration.mdx [hunk 1]
```diff
@@ -181,10 +181,10 @@ sidebar: [
 type SidebarItem = {
 	label: string;
 	translations?: Record<string, string>;
+  badge?: string | BadgeConfig;
```
Comment (100gle) [inline docs/src/content/docs/zh-cn/reference/configuration.mdx:184]:
```suggestion
type SidebarItem = {
	label: string;
	translations?: Record<string, string>;
    badge?: string | BadgeConfig;
```

缩进没对齐

File: docs/src/content/docs/zh-cn/guides/sidebar.mdx [hunk 2]
```diff
@@ -249,6 +249,12 @@ starlight({
 				},
 			],
 		},
+    // 一个带有 "Deprecated" 徽章的自动生成分组。
+		{
+			label: 'Reference',
+			badge: 'Deprecated',
+			autogenerate: { directory: 'reference' },
+		},
```
Comment (100gle) [inline docs/src/content/docs/zh-cn/guides/sidebar.mdx:257]:
```suggestion
        // 一个带有 "Deprecated" 徽章的自动生成分组。
		{
			label: 'Reference',
			badge: 'Deprecated',
			autogenerate: { directory: 'reference' },
		},
```

缩进没对齐

**Review by 100gle — CHANGES_REQUESTED** [review]

**Review by 100gle — APPROVED** [review]
LGTM

**Review by delucis — APPROVED** [review]
Looks great!

Just for your information, you could also have chosen to translate the badges and label text. This works too, so I’ll merge this, but if you ever want to revisit it, feel free!

**Pull request merged**